### ISSUE TITLE
Remove or improve redundant descriptions in workflow editor

### ISSUE THREAD

**Issue by solth** [kitodo/kitodo-production#4786]
The panel containing the task properties in the workflow editor currently has descriptions only for two or those properties - "Metadata" and "Automatic". Those descriptions do not offer any additional value, though, and are very redundant
![Bildschirmfoto 2021-10-22 um 09 30 56](https://user-images.githubusercontent.com/19183925/138413196-287368de-23ea-4369-9820-078f6906ba60.png)
It's also very inconsistent that only two of the properties have a description text and the others have not!
I therefore suggest to either
a) remove the existing descriptions or
b) improve them and add (better) descriptions to the other properties as well

**Comment by solth** [comment]
Dear @Erikmitk can you remove these redundant descriptions? Can I assign this issue to you?

**Comment by Erikmitk** [comment]
> Dear [@Erikmitk](https://github.com/Erikmitk) can you remove these redundant descriptions? Can I assign this issue to you?

Yes, sure. I'm currently working on the editor again to get it to built again so that would fit right in.
